Create dsct-tr-to-pr

pwin opened the pull request:
draft transition request from Candidate Recommendation to Proposed Recommendation

pwin committed:
Create dsct-tr-to-pr

draft transition request from Candidate Recommendation to Proposed Recommendation

pwin committed:
Rename dsct-tr-to-pr to dcat-tr-to-pr

kcoyle reviewed (commented):
It looks to me like this is what is still needed to complete this:

line 30 - summary of changes since CR

line 42 - statement about additional feedback, e.g. the recent github issues

line 54 - say that no further issues will be in DCAT2, give github label for future work

line 64 - has everything been said about features at risk?

pwin committed:
Update dcat-tr-to-pr

adding text about future work and including link to github 'future work'

riccardoAlbertoni committed:
provided a summary of changes

riccardoAlbertoni commented:
I have added to "line 30 - summary of changes since CR".
 Perhaps, the list of changes is too detailed, feel free to change it as you like.

riccardoAlbertoni commented on dcat/dcat-tr-to-pr:
```diff
@@ -0,0 +1,80 @@
+ - [PR 1127](https://github.com/w3c/dxwg/pull/1127) has uniformed the use of blanks and punctuation in the literals included in dcat.ttl;
+ - [PR 1135]([https://github.com/w3c/dxwg/pull/1135](https://github.com/w3c/dxwg/pull/1135)) has fixed coordinate order in ga-courts example;
+ - [PR 1136](https://github.com/w3c/dxwg/pull/1136) has inserted a note in section DCAT History to clarify that DCAT 2 maintains the DCAT namespace as it preserves backward compatibility with DCAT;
+ - [PR 1140](https://github.com/w3c/dxwg/pull/1140) has revised some class descriptions;
```
```suggestion
 - [PR 1140](https://github.com/w3c/dxwg/pull/1140) has revised some class descriptions. It has removed descriptions of dcat:accessRights and dct:license for dcat:DataServices. These descriptions were redundant as they are already provided in the class dcat:Resource from which the two properties are inherited;
```

riccardoAlbertoni committed:
Update dcat/dcat-tr-to-pr

Adding explanation on duplicated descriptions for dcat:accessRights and dct:license in DataService

riccardoAlbertoni committed:
Merge branch 'gh-pages' into dcat-tr-to-pr

pwin committed:
Update dcat-tr-to-pr

pwin committed:
Update dcat-tr-to-pr

pwin committed:
@@ report on any additional feedback

adding a note about the discussion on "conforms to"
